module/plugins/accounts/MegaCoNz.py

```diff
@@ -118,7 +129,7 @@ def get_password_key(self, password):
 
         return MegaCrypto.str_to_a32(password_key)
 
-    def get_user_hash(self, user, password_key):
+    def get_user_hash_v1(self, user, password_key):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/pyLoad/pyload/pullRequest?prid=5638731)